apps/testclient/management/commands/create_test_users_and_applications_batch.py

```diff
@@ -64,10 +82,14 @@ def create_dev_users_apps_and_bene_crosswalks(group, bene_count, dev_count, app_
     count = 0
     file_cnt = 0
     synthetic_bene_cnt = 0
+    begin_number = get_first_available_number("bene")
+
     for f in files:
         print("file={}".format(f))
         bene_rif = open('./synthetic-data/{}'.format(f), 'r')
```

Comment:
is this synthetic data the same data in the sandbox environment?

Reply:
I don't believe they're the exact same. The way BFD has generated synthetic data has evolved through time and this is synthetic data that was generated around the time that this command was originally written.